config/datasets/RADCURE.yaml

```
@@ -8,32 +8,32 @@ CLINICAL:
         time_label: "Length FU"
         event_label: "Status"
         convert_to_years: False
-        event_value_mapping: {0: ['Alive'], 1: ['Dead']}
-    EXCLUSION_VARIABLES: {'Ds Site': ['Sarcoma', 'Unknown', 'Paraganglioma', 'Salivary Glands', 'Other', 'benign tumor', 'Orbit', 'Lacrimal gland', 'Skin']}
+        event_value_mapping: {"Alive": 0, "Dead": 1}
+    EXCLUSION_VARIABLES: {} #{'Ds Site': ['Sarcoma', 'Unknown', 'Paraganglioma', 'Salivary Glands', 'Other', 'benign tumor', 'Orbit', 'Lacrimal gland', 'Skin']}
+    INCLUSION_VARIABLES: {'Ds Site': ['Oropharynx'],
+                          'HPV': ['Yes, positive', 'Yes, Negative']
+                         }
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 25

(indentation)